lib/github_processor.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class GithubProcessor
+  def initialize(event_sender)
+    @event_sender = event_sender
+  end
+
+  def process(json_body)
+    event = GithubMessageFactory.build(json_body)
+
```

Comment:
Should there be an `event.nil?` short circuit here? I don't know if the message factory supports all the different types it will be getting sent.